src/clj/chronograph/middleware.clj

```diff
@@ -81,7 +81,7 @@
   [handler roles]
   (fn [{:keys [user organization] :as request}]
     (jdbc/with-transaction [tx db/datasource]
-      (let [user-role  (acl/role tx user organization)]
+      (let [user-role  (acl/role tx (:users/id user) (:organizations/id organization))]
```

Comment:
Nonbinding suggestion:

Alternate destructuring in the `fn` to make it obvious that user and organization are maps, and not scalars:

```clojure
{{user-id :users/id} :user
 {org-id :organizations/id} :organization
 :as request
```